### PR TITLE
[Kotlin Migration] AdaptionUtil 

### PR DESCRIPTION
**Pull request by codingtosh**

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.kt [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+ * the terms of the GNU General Public License as published by the Free Software        *
+ * Foundation; either version 3 of the License, or (at your option) any later           *
+ * version.                                                                             *
+ * *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY      *
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A      *
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.             *
+ * *
+ * You should have received a copy of the GNU General Public License along with         *
+ * this program.  If not, see <http:></http:>//www.gnu.org/licenses/>.                           *
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.kt:14]: copyright was corrupted

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
copyright issue, didn't look further

File: AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.java [hunk 2]
```diff
@@ -1,182 +0,0 @@
...
-                || isIntentResolved(ctx, new Intent().setComponent(new ComponentName("com.miui.securitycenter", "com.miui.permcenter.autostart.AutoStartManagementActivity")))
-                || isIntentResolved(ctx, new Intent("miui.intent.action.POWER_HIDE_MODE_APP_LIST").addCategory(Intent.CATEGORY_DEFAULT))
-                || isIntentResolved(ctx, new Intent().setComponent(new ComponentName("com.miui.securitycenter", "com.miui.powercenter.PowerSettings")));
-    }
-
-
-    @SuppressWarnings( {"unused", "RedundantSuppression"})
-    public static boolean shouldCurrentUserBuyDifferentPhone() {
-        return isRunningMiui();
-    }
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.java:171]: Probably for the best 😉

**Review by david-allison-1 — APPROVED** [review]
LGTM

File: AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.java (hunk 2 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.java:171]: Haha, nice easter egg, too bad it didn't remain in. 😄

Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.java:171]: biggest irony: @david-allison-1 just bought a MIUI phone on purpose (to help debug crap MIUI issues, but...what masochism haha)

**Review by mikehardy — APPROVED** [review]
nice

**Pull request merged**